recipes/mkl_random/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.1" %}
+{% set buildnumber = 0 %}
+
+
+package:
+    name: mkl_random
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/IntelPython/mkl_random
```

Comment:
Please build from tarballs https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos